### DISCUSSION TITLE
Baseline install instructions

### DISCUSSION THREAD

**Discussion by Gasman2014** [aidanmacgregor/EE_WiFi-BT_WiFi-Autologin-OpenWRT#5] [Announcements]
I'm interested in your project but, although I'm reasonably IT literate, I'm not a network expert and I'm also new to OpenWRT. I've successfully installed OpenWRT on an old router and added you EE login script, edited the .rc file as required and started the service. I found I needed to add a radio0 instance but got somewhat lost in the subsequent configuration. After an hour or two and multiple restarts,  I have made only a little progress. Although I have an activated radio0 and a running service, I am not connecting successfully and  not sure how to proceed. Could you perhaps add a little extra to the Read.me to explain how you have configured the router settings for a baseline, functional setup as per your diagram? In particular, the wifi is disabled on a vanilla install of OpenWRT. It is not completely clear from your instructions as to what the initial setup is and if there is any need for further modules - for instance, do I need to install the relayd service, do how do I configure my radio0 device which is not enabled on the initial install, do I need to adjust the LAN settings on lan0? Perhaps direction to a suitable page or simple configuration options in the OpenWRT instructions would help? 

Are there any OpenWRT wizards that can get this set up quickly with some sensible defaults. Whilst OpenWRT is undoubtedly very powerful and configurable, it is also rather daunting and frustrating when I just want to get a service up and running quickly.

**Comment by Gasman2014** [comment]
With a bit more work, I have this working. Can I suggest that you link this page into he Read.me as it is an essential prerequisite to getting up and running?

https://openwrt.org/docs/guide-user/network/wifi/connect_client_wifi

**Comment by aidanmacgregor** [comment]
> With a bit more work, I have this working. Can I suggest that you link this page into he Read.me as it is an essential prerequisite to getting up and running?
> 
> https://openwrt.org/docs/guide-user/network/wifi/connect_client_wifi

yhea a good idea, also want to get a video made :)
